src/poliastro/twobody/propagation.py

```diff
@@ -188,6 +195,868 @@ def _kepler(k, r0, v0, tof, *, numiter):
     return r, v
 
 
+def mikkola(k, r, v, tofs, rtol=None):
+    """ Solves Kepler Equation by a cubic approximation. This method is valid
+    no mater the orbit's nature.
+
+    Parameters
+    ----------
+    k : ~astropy.units.Quantity
+        Standard gravitational parameter of the attractor.
+    r : ~astropy.units.Quantity
+        Position vector.
+    v : ~astropy.units.Quantity
+        Velocity vector.
+    tofs : ~astropy.units.Quantity
+        Array of times to propagate.
+    rtol: float
+        This method does not require of tolerance since it is non iterative.
+
+    Returns
+    -------
+    rr : ~astropy.units.Quantity
+        Propagated position vectors.
+    vv : ~astropy.units.Quantity
+
+    Note
+    ----
+    This method was derived by Seppo Mikola in his paper *A Cubic Approximation
+    For Kepler's Equation* with DOI: https://doi.org/10.1007/BF01235850
+    """
+
+    k = k.to(u.m ** 3 / u.s ** 2).value
+    r0 = r.to(u.m).value
+    v0 = v.to(u.m / u.s).value
+    tofs = tofs.to(u.s).value
+
+    results = [_mikkola(k, r0, v0, tof) for tof in tofs]
+    return (
+        [result[0] for result in results] * u.m,
+        [result[1] for result in results] * u.m / u.s,
+    )
+
+
+def _mikkola(k, r0, v0, tof, rtol=None):
+    """ Raw algorithm for Mikkola's Kepler solver.
+
+    Parameters
+    ----------
+    k : ~astropy.units.Quantity
+        Standard gravitational parameter of the attractor.
+    r : ~astropy.units.Quantity
+        Position vector.
+    v : ~astropy.units.Quantity
+        Velocity vector.
+    tofs : ~astropy.units.Quantity
+        Array of times to propagate.
+    rtol: float
+        This method does not require of tolerance since it is non iterative.
+
+    Returns
+    -------
+    rr : ~astropy.units.Quantity
+        Propagated position vectors.
+    vv : ~astropy.units.Quantity
+
+    Note
+    ----
+    Original paper: https://doi.org/10.1007/BF01235850
+    """
+
+    # Solving for the classical elements
+    p, ecc, inc, raan, argp, nu = rv2coe(k, r0, v0)
+    if ecc >= 1.0:
+        raise ValueError("Mikkola only works with elliptical orbits.")
+    M0 = nu_to_M(nu, ecc)
```

Comment:
I have a question though: a few of the new propagators use `nu_to_M`. Isn't it easier to keep `delta` and do `nu_to_M(nu, ecc, delta=0)` in these cases? That way, perhaps we can retain the behavior of `mean_motion` and making all the rest do the same. What do you think?

Reply:
Sure! I have checked and by keeping delta, `mean_motion` works without problem for any `ecc` regime :+1: